decidim/decidim
# Add order position in conferences

#### :tophat: What? Why?

For making it consistent with other spaces, this PR adds the weight feature (aka "Order position") in Conferences.

This is related to #9509, where we used the date for ordering, but I think it's more consistent with other spaces and gives more control to admins to be able to change this. 

#### :pushpin: Related Issues

- Fixes #10064
- Related to #10062

#### Testing

1. Try the weight field in conferences
2. See how it changes the order 
 
:hearts: Thank you!
